src/KubernetesClient.Models/IItems.cs

```diff
@@ -12,4 +12,12 @@ public interface IItems<T>
         /// </summary>
         IList<T> Items { get; set; }
     }
+
+    public static class ItemsExt
+    {
+        public static IEnumerator<T> GetEnumerator<T>(this IItems<T> items)
+        {
+            return items.Items.GetEnumerator();
```

Comment:
Just curious, is it possible for `items.Items` to be null in any case? What is the expected behavior? Let it throw or gracefully quit the enumeration?

Reply:
yup you are right, would throw npe
maybe make it empty would be better

the issue now is that linq is not compatible with this impl